src/api/v1beta1/properties_test.go

```diff
@@ -417,3 +420,85 @@ func TestGetOneAgentEnvironment(t *testing.T) {
 		assert.Len(t, env, 0)
 	})
 }
+
+func TestDynaKube_ShallUpdateActiveGateConnectionInfo(t *testing.T) {
+	dk := DynaKube{
+		Status: DynaKubeStatus{
+			DynatraceApi: DynatraceApiStatus{
+				LastTokenScopeRequest:               metav1.Time{},
+				LastOneAgentConnectionInfoRequest:   metav1.Time{},
+				LastActiveGateConnectionInfoRequest: metav1.Time{},
+			},
+		},
+	}
+
+	timeProvider := timeutil.NewProvider()
+	tests := map[string]struct {
+		lastRequestTimeDeltaMinutes int
+		shallUpdate                 bool
```

Comment:
```suggestion
		update                 bool
```

Reply:
Here the `shall` is justified, because it reads "expect update" not "do that update"